backend/src/apiserver/server/experiment_server_test.go

```diff
@@ -394,7 +395,7 @@ func TestListExperiment_Multiuser(t *testing.T) {
 		} else {
 			if err != nil {
 				t.Errorf("TestListExperiment_Multiuser(%v) expect no error but got %v", tc.name, err)
-			} else if !cmp.Equal(tc.expectedExperiments, response.Experiments) {
+			} else if !cmp.Equal(tc.expectedExperiments, response.Experiments, cmpopts.IgnoreFields(api.Experiment{}, "CreatedAt")) {
```

Comment:
Issues due to: 
```panic: cannot handle unexported field at```  possible solution is:
- Use custom Comparer( thought it was to much work).
- If you control the implementation of type, you can also consider using an Exporter( we don't control it) .
- Avoid the fields in the comparison( seems like the simples solution and what I decided to do)